pkg/apis/zookeeper/v1beta1/zookeepercluster_types.go

```diff
@@ -383,17 +387,38 @@ func (c *ZookeeperConfig) withDefaults() (changed bool) {
 	return changed
 }
 
+type Storage struct {
+	//storagetype is used to tell which type of storage we will be using
+	//It can take either Ephemeral or persistence
+	//Default Storagetype is Persistence storage
+	Storagetype string `json:"storagetype,omitempty"`
```

Comment:
since we follow camel case, please rename this to `StorageType`

Reply:
Done.